packages/client/src/users/closets/getUserCloset.ts

```diff
@@ -0,0 +1,29 @@
+import { adaptError } from '../../helpers/client/formatError.js';
+import client from '../../helpers/client/index.js';
+import join from 'proper-url-join';
+import type { GetUserCloset } from './types/index.js';
+
+/**
+ * Method responsible for getting a specific user closet.
```

Comment:
As discussed in slack it seems that this is not entirely correct, the first draft of the spike implied this but this endpoint currently returns the items of a specific closet to be rendered.

Reply:
Fixed. Changed the description and changed the names of actions, clients and types to reflect what we talked privately. `fetchUserCloset` -> `fetchUserClosetItems`